downstairs/src/region.rs

```diff
@@ -1515,7 +1554,7 @@ impl Region {
                 };
 
             if computed_hash != write.hash {
-                println!("Failed write hash validation");
+                println!("Failed write hash validation {}", computed_hash);
```

Comment:
if we're putting in the hash value, let's put more context in here: at least offset, computed hash, and write.hash

Reply:
That was really a debug additin, so I could print the hash and then hard code the result in the test.  I'll remove it.